### ISSUE TITLE
DEBUG_PKG does not work when project is not on drive C: on Windows

### ISSUE THREAD

**Issue by FlorentMasson** [vercel/pkg#1207]
I get this error after I follow the instructions [here](https://github.com/vercel/pkg#advanced)

```
D:\project\server_exe>server-win.exe
------------------------------- virtual file system
C:\snapshot
pkg/prelude/bootstrap.js:322
  if (error) throw error;
             ^

Error: Directory 'C:\**\' was not included into executable at compilation stage. Please recompile adding it as asset or script.
    at error_ENOENT (pkg/prelude/bootstrap.js:539:17)
    at readdirFromSnapshot (pkg/prelude/bootstrap.js:1057:29)
    at Object.readdirSync (pkg/prelude/bootstrap.js:1083:19)
    at dumpLevel (pkg/prelude/bootstrap.js:2066:18)
    at installDiagnostic (pkg/prelude/bootstrap.js:2111:23)
    at pkg/prelude/bootstrap.js:2142:3
    at pkg/prelude/bootstrap.js:2144:3
    at readPrelude (internal/bootstrap/pkg.js:31:12)
    at internal/bootstrap/pkg.js:36:18
    at internal/bootstrap/pkg.js:43:4 {
  errno: -4058,
  code: 'ENOENT',
  path: 'C:\\snapshot',
  pkg: true
}
```

It works fine when I copy the same project to C:

**Comment by erossignon** [comment]
@FlorentMasson  can you provide a minimalist test that reproduces the behavior ? 
I try compiling a simple test script and move it to a different drive but I cannot reproduce it.

**Comment by FlorentMasson** [comment]
Is this enough as a minimalist test?
To clarify, I'm compiling from a different drive (D:), not on C: then moving the executable

```
D:\>pkg -v
5.2.1

D:\>md testpkg

D:\>cd testpkg

D:\testpkg>echo console.log('hello'); > test.js

D:\testpkg>pkg -d test.js
> pkg@5.2.1
> Targets not specified. Assuming:
  node12-linux-x64, node12-macos-x64, node12-win-x64
> [debug] Bytecode of D:\testpkg\test.js is added to queue.
> [debug] Stat info of D:\testpkg\test.js is added to queue.
> [debug] Directory D:\testpkg is added to queue.
> [debug] Stat info of D:\testpkg is added to queue.
> [debug] Directory D:\ is added to queue.
> [debug] Stat info of D:\ is added to queue.
> [debug] The file was included as bytecode (no sources)
  D:\testpkg\test.js
> [debug] The file was included as bytecode (no sources)
  D:\testpkg\test.js
> [debug] files & folders deduped =
  test.js
> [debug] The directory files list was included (1 item)
  D:\testpkg
> [debug] The directory files list was included (1 item)
  D:\testpkg
> [debug] files & folders deduped =
  testpkg
> [debug] The directory files list was included (1 item)
  D:\
> [debug] The directory files list was included (1 item)
  D:\
> [debug] Targets:
  [
  {
    "nodeRange": "node12",
    "platform": "linux",
    "arch": "x64",
    "output": "D:\\testpkg\\test-linux",
    "forceBuild": false,
    "fabricator": {
      "nodeRange": "node12",
      "platform": "win",
      "arch": "x64",
      "binaryPath": "C:\\Users\\florent\\.pkg-cache\\v3.1\\fetched-v12.22.1-win-x64"
    },
    "binaryPath": "C:\\Users\\florent\\.pkg-cache\\v3.1\\fetched-v12.22.1-linux-x64"
  },
  {
    "nodeRange": "node12",
    "platform": "macos",
    "arch": "x64",
    "output": "D:\\testpkg\\test-macos",
    "forceBuild": false,
    "fabricator": {
      "nodeRange": "node12",
      "platform": "win",
      "arch": "x64",
      "binaryPath": "C:\\Users\\florent\\.pkg-cache\\v3.1\\fetched-v12.22.1-win-x64"
    },
    "binaryPath": "C:\\Users\\florent\\.pkg-cache\\v3.1\\fetched-v12.22.1-macos-x64"
  },
  {
    "nodeRange": "node12",
    "platform": "win",
    "arch": "x64",
    "output": "D:\\testpkg\\test-win.exe",
    "forceBuild": false,
    "fabricator": {
      "nodeRange": "node12",
      "platform": "win",
      "arch": "x64",
      "binaryPath": "C:\\Users\\florent\\.pkg-cache\\v3.1\\fetched-v12.22.1-win-x64"
    },
    "binaryPath": "C:\\Users\\florent\\.pkg-cache\\v3.1\\fetched-v12.22.1-win-x64"
  }
]

D:\testpkg>test-win.exe
hello

D:\testpkg>set DEBUG_PKG=1

D:\testpkg>test-win.exe
------------------------------- virtual file system
C:\snapshot
pkg/prelude/bootstrap.js:322
  if (error) throw error;
             ^

Error: Directory 'C:\**\' was not included into executable at compilation stage. Please recompile adding it as asset or script.
    at error_ENOENT (pkg/prelude/bootstrap.js:539:17)
    at readdirFromSnapshot (pkg/prelude/bootstrap.js:1057:29)
    at Object.readdirSync (pkg/prelude/bootstrap.js:1083:19)
    at dumpLevel (pkg/prelude/bootstrap.js:2066:18)
    at installDiagnostic (pkg/prelude/bootstrap.js:2111:23)
    at pkg/prelude/bootstrap.js:2142:3
    at pkg/prelude/bootstrap.js:2144:3
    at readPrelude (internal/bootstrap/pkg.js:31:12)
    at internal/bootstrap/pkg.js:36:18
    at internal/bootstrap/pkg.js:43:4 {
  errno: -4058,
  code: 'ENOENT',
  path: 'C:\\snapshot',
  pkg: true
}

D:\testpkg>
```

**Comment by erossignon** [comment]
fixed in 5.3.0

**Comment by hughdev** [comment]
same problem in pkg version 5.3.1
 Error: File or directory 'Z:\**\' was not included into executable at compilation stage. Please recompile adding it as asset or script.
     at error_ENOENT (pkg/prelude/bootstrap.js:557:19)
     at findNativeAddonForStat (pkg/prelude/bootstrap.js:1271:32)
     at statFromSnapshot (pkg/prelude/bootstrap.js:1294:25)
     at Object.statSync (pkg/prelude/bootstrap.js:1308:12)
     at C:\snapshot\test\local-space.service.js
     at Array.forEach (<anonymous>)
     at LocalSpaceService.getLocalFSInfo (C:\snapshot\test\local-space.service.js)
     at runMicrotasks (<anonymous>)
     at processTicksAndRejections (internal/process/task_queues.js:97:5)
     at async LocalSpaceController.fun (C:\snapshot\test\local-space.controller.js) {
   errno: -4058,
   code: 'ENOENT',
   path: 'Z:\\snapshot',
   pkg: true
 }
